oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/StorageEsInstaller.java

```diff
@@ -84,8 +85,14 @@ public boolean isExists(Model model) throws StorageException {
                 Optional<Index> index = esClient.getIndex(tableName);
                 Mappings historyMapping = index.map(Index::getMappings).orElseGet(Mappings::new);
                 structures.putStructure(tableName, historyMapping, index.map(Index::getSettings).orElseGet(HashMap::new));
-                exist = structures.containsMapping(tableName, createMapping(model))
-                    && structures.compareIndexSetting(tableName, createSetting(model));
+                boolean containsMapping = structures.containsMapping(tableName, createMapping(model));
+                // Do not check index settings in the "no-init mode",
+                // because the no-init mode OAP server doesn't take responsibility for index settings.
```

Comment:
Does no init mode take responsibility of index mappings?

Reply:
no-init mode needs check and wait init mappings

Reply:
Kind of. As OAP should wait for the index mappings contain the required.